README.md

```diff
@@ -55,22 +55,60 @@ Switch to **Raw JSON** mode and add the following to your `"modules"` array:
 
 Replace the value of the "version" field with the value you determined above with the docker pull command.
 
-Add the following to your `"modules"` array:
+Add the following to your `"services"` array:
 
 ```json
 {
-  "type": "registry",
-  "name": "viam_mlmodelservice-triton-jetpack",
-  "module_id": "viam:mlmodelservice-triton-jetpack",
-  "version": "0.4.0"
+ "name": "my-triton-module",
+  "type": "mlmodel",
+  "namespace": "rdk",
+  "model": "viam:mlmodelservice:triton"
 }
 ```
 
+The model will now be configured with a card like the following:
+
+![The triton service card in the Viam app config builder, showing deployment options.](https://docs.viam.com/registry/triton/triton-config-builder.png)
+
+Note that the parameters shown, `"model_path"`, `"label_path"`, and `"num_threads"` are not applicable for this module, and should be left blank.
+Ignore this card's interface and move to [creating a model repository](#create-a-repository-to-store-the-ml-model-to-deploy).
+
 > [!NOTE]  
 > For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
 
+
+### Create a repository to store the ML model to deploy
+
+Currently, you must manually create a Triton [model repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html).
+On your robot's Jetson computer, create a [structured repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html) under the `~/.viam` directory.
+The exact subpath under `~/.viam` does not matter.
```

Comment:
I tried this today, and it didn't work unless I put it on my Orin's `/root/.viam` directory. I think it depends from which `.viam` directory the user runs from.

Reply:
Thanks for pointing this out, I added a note about the placement of the model repository. Happy to make further changes.